ee/localserver/server.go

```diff
@@ -461,6 +457,12 @@ func (ls *localServer) presenceDetectionHandler(next http.Handler) http.Handler
 			)
 		}
 
+		if !ls.presenceDetectionMutex.TryLock() {
+			http.Error(w, "presence detection already in progress", http.StatusTooManyRequests)
```

Comment:
Should this be an error? Maybe?

Reply:
I think it's best to leave it as an error for because if we got here, the request is asking for presence detection which we cannot give at this time.